pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,164 @@
+    code with to ensure if it actually errors.
+
+    If your code runs fine, you could try restarting your editor. If you have newly installed a module it may just not
+    have detected it yet. If that doesn't help, you may need to configure your editor to ensure it is looking for the
+    module in the correct Python environment. See [our guide on virtual
+    environments](../installing-and-using-python/#virtual-environments) for more information.
+
+-   #### Was the install successful?
+
+    Look out for errors when installing the module you want. If you get an error, that's probably why the import isn't
+    working, so you should look into it. Often a Google search of the relevant error text will help.
+
+-   #### Did you use the correct module name?
+
+    Double check that you haven't made a typo in the name you are importing, or in what you installed from
+    [PyPI](https://pypi.org/), you need to make sure you type the name exactly as it should be.
+
+    Also, **the name you should `import` may not be the same as the name you `pip install`**. Check the module's
+    documentation or PyPI page if you are unsure. Examples of this are:
+
+    -   [**opencv-python**](https://pypi.org/project/opencv-python/): You need to `pip install opencv-python`, but the
+        import has to be `import cv2`
+    -   [**discord.py**](https://pypi.org/project/discord.py/): You need to `pip install discord.py`, but the import has
+        to be `import discord`
+    -   [**python-dotenv**](https://pypi.org/project/python-dotenv/): You need to `pip install python-dotenv`, but the
+        import has to be `import dotenv`
+    -   [**Pillow**](https://pypi.org/project/Pillow/): You need to `pip install Pillow`, but the import is
+        usually `from PIL import Image`.
+
+    Normal imports in Python can't have dashes (`-`) in them as it would be interpreted as subtraction. Though [there
+    are workarounds](https://stackoverflow.com/questions/7583652/python-module-with-a-dash-or-hyphen-in-its-name).
+
+-   #### Are you installing to the same environment you're running your code from?
+
+    This is a very common issue. When you install a module from PyPI, you will install it into a single Python
+    environment, the one that the `pip` you invoked is part of. This could be different to the one you are running the
+    script from if it's using a different Python installation, or a virtual environment.
+
+    ##### I'm using PyCharm
+
+    > PyCharm creates a virtual environment for each project by default, so if you're installing a module using Pycharm
+    > you need to make sure you also run your code through PyCharm. This also applies the other way round, if you want
+    > to use a module in PyCharm you need to install it through PyCharm.
+    >
+    > It's also possible to change PyCharm to use your system environment, see [their guide on configuring your
```

Comment:
We can add a section here that helps people identify the currently active virtual environment in PyCharm.
If you look at the bottom right, you will see your project root's name between parenthesis along with the python version like this.
![image](https://user-images.githubusercontent.com/48383734/200617946-ad54c699-b29c-4582-a404-5d9efb7d8848.png)

If you click on that, you can see the full path to your virtual env here
![image](https://user-images.githubusercontent.com/48383734/200618479-1306991f-ecdb-4a01-8a57-e82b54719838.png)

So if you want to install a package into that one from the terminal in Pycharm, you might want to make sure that you have the correct one active by checking the venv name between parenthesis here
![image](https://user-images.githubusercontent.com/48383734/200618764-48e1a10f-d313-4fcd-96fd-96b70e62f828.png)

I remember making this mistake when I was first starting

Reply:
Sure. I added some images and info mentioning these.